src/lib.rs

```diff
@@ -6,6 +6,7 @@
 //! SECurity) service. For a more in-depth guide of the system architecture,
 //! supported operations and other Parsec-related topics, see our
 //! [Parsec Book](https://parallaxsecond.github.io/parsec-book/index.html).
+#![macro_use]
```

Comment:
Is this needed to use the macro? I thought that with recent versions of `rustc` you could just import the macro name as a function.

Reply:
Indeed! I've removed it, thought it was deprecated only for importing